pkgs/development/compilers/llvm/14/llvm/default.nix

```diff
@@ -129,6 +129,9 @@ in stdenv.mkDerivation (rec {
   # E.g. mesa.drivers use the build-id as a cache key (see #93946):
   LDFLAGS = optionalString (enableSharedLibraries && !stdenv.isDarwin) "-Wl,--build-id=sha1";
 
+  # workaround https://github.com/NixOS/nixpkgs/issues/201254
+  NIX_LDFLAGS = optionalString (stdenv.isLinux && stdenv.isAarch64 && stdenv.cc.isGNU) "-lgcc";
+
```

Comment:
I suppose this change might be the reason that this would cause so many rebuilds, even on Darwin which should really not be effected at all.

Should I change that to add the `NIX_LDFLAGS` attribute optionally?

Reply:
You can avoid rebuilds by using
```nix
{
  NIX_LDFLAGS = if stdenv.isLinux && stdenv.isAarch64 && stdenv.cc.isGNU then "-lgcc" else null;
}
```

Reply:
Thank you, @vcunat! Changed.